src/__experimental__/components/numeral-date/numeral-date.d.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
```

Comment:
You forgot to update the `NumeralDateProps` interface after applying my requested changes :)

Reply:
Well spotted.

Reply:
Resolved in the latest commit.